connect/json/src/main/java/org/apache/kafka/connect/json/JsonConverter.java

```diff
@@ -361,6 +367,22 @@ private SchemaAndValue jsonToConnect(JsonNode jsonValue) {
         return new SchemaAndValue(schema, convertToConnect(schema, jsonValue.get(JsonSchema.ENVELOPE_PAYLOAD_FIELD_NAME)));
     }
 
+    private SchemaAndValue schemalessJsonToConnect(JsonNode jsonValue) {
+        if (jsonValue == null)
+            return SchemaAndValue.NULL;
```

Comment:
Nit: we always use braces around if-blocks.

Reply:
(Always)[https://github.com/apache/kafka/blob/trunk/connect/json/src/main/java/org/apache/kafka/connect/json/JsonConverter.java#L354-L355]? ;)

I'll apply that rule for the whole file in a followup commit.